Python/import.c

```diff
@@ -1135,8 +1128,10 @@ find_frozen(PyObject *modname)
         PyErr_Clear();
         return NULL;
     }
-    if (!use_frozen(name)) {
-        return NULL;
+    if (!use_frozen()) {
+        if (!is_essential_frozen_module(name)) {
```

Comment:
I'd use a single `if` and combine the conditions with `&&`.

Reply:
fixed